tools/toggle-benchmark.sh

```
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+until [ "$(busctl --user get-property org.gnome.Shell /org/gnome/Shell/Extensions/ddterm com.github.amezin.ddterm.Extension HasWindow)" = "b true" ]
+do
+    busctl --user wait /org/gnome/Shell/Extensions/ddterm org.freedesktop.DBus.Properties PropertiesChanged
+done &
+
+~/.local/share/gnome-shell/extensions/ddterm@amezin.github.com/bin/com.github.amezin.ddterm
+
+wait $(jobs -p)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)